wolfcrypt/src/sp_int.c

```diff
@@ -17888,12 +17878,43 @@ int sp_read_unsigned_bin(sp_int* a, const byte* in, word32 inSz)
     #endif /* LITTLE_ENDIAN_ORDER */
         }
 #endif
-        sp_clamp_ct(a);
+        if (! ct)
```

Comment:
No space after '!'

Reply:
fixed (completely removed the conditionalization of the clamping, as discussed).